client/sandbox.go

```diff
@@ -81,37 +86,33 @@ func (s *sandboxClient) Labels(ctx context.Context) (map[string]string, error) {
 	return sandbox.Labels, nil
 }
 
-func (s *sandboxClient) Start(ctx context.Context) error {
-	resp, err := s.client.SandboxController(s.metadata.Sandboxer).Start(ctx, s.ID())
-	if err != nil {
-		return err
-	}
+func (s *sandboxClient) Status(ctx context.Context, verbose bool) (api.ControllerStatus, error) {
+	return s.client.SandboxController(s.metadata.Sandboxer).Status(ctx, s.metadata.ID, verbose)
+}
 
-	s.pid = &resp.Pid
-	return nil
+func (s *sandboxClient) Platform(ctx context.Context) (platforms.Platform, error) {
+	return s.client.SandboxController(s.metadata.Sandboxer).Platform(ctx, s.ID())
 }
 
-func (s *sandboxClient) Wait(ctx context.Context) (<-chan ExitStatus, error) {
-	c := make(chan ExitStatus, 1)
-	go func() {
-		defer close(c)
+func (s *sandboxClient) Create(ctx context.Context, opts ...api.CreateOpt) error {
+	return s.client.SandboxController(s.metadata.Sandboxer).Create(ctx, s.metadata, opts...)
+}
 
-		exitStatus, err := s.client.SandboxController(s.metadata.Sandboxer).Wait(ctx, s.ID())
-		if err != nil {
-			c <- ExitStatus{
-				code: UnknownExitStatus,
-				err:  err,
-			}
-			return
-		}
+func (s *sandboxClient) Start(ctx context.Context) (api.ControllerInstance, error) {
+	return s.client.SandboxController(s.metadata.Sandboxer).Start(ctx, s.ID())
+}
 
-		c <- ExitStatus{
-			code:     exitStatus.ExitStatus,
-			exitedAt: exitStatus.ExitedAt,
+func (s *sandboxClient) Wait(ctx context.Context) (<-chan api.ExitStatus, error) {
+	ch := make(chan api.ExitStatus, 1)
+	go func() {
+		status, err := s.client.SandboxController(s.metadata.Sandboxer).Wait(ctx, s.ID())
```

Comment:
suggest to close `ch` in defer and remain that `UnknownExitStatus` if Wait returns error.

Reply:
done